### PR TITLE
#13 Preview page

### LINKED ISSUE #46: hide google maps api key and db access keys
- [x] make it into environment variable
- [x] modify readme file as well

### PR DESCRIPTION
**Pull request by polarbearTW**
-Made 2 pages: Landing page (for non logged in users) & Home page (logged in users)
-Hide API key from the code
-Set username & password on httpd configuration file
-A GMap with default markers on landing page
-Remove log in button from header

### CODE REVIEW

**Comment by hjusassac** [comment]
could you edit the readme file as well,  please?

**Comment by polarbearTW** [comment]
> could you edit the readme file as well, please?

Hi Hyeonju, I just did, please check, thanks!

**Review by LEMON4DE — DISMISSED** [review]

File: public/styles.css [hunk 1]
```diff
@@ -107,6 +107,26 @@ header {
     text-decoration: underline;
 }
 
+<<<<<<< HEAD
```
Comment (LEMON4DE) [inline public/styles.css]:
```suggestion
```

File: public/styles.css [hunk 2]
```diff
@@ -107,6 +107,26 @@ header {
...
+  /* 
+   * Optional: Makes the sample page fill the window. 
+   */
+  html,
+  body {
+    height: 100%;
+    margin: 0;
+    padding: 0;
+  }
+=======
```
Comment (LEMON4DE) [inline public/styles.css]:
```suggestion
```

File: public/styles.css [hunk 3]
```diff
@@ -127,4 +147,5 @@ header {
     padding: 10px;
     background-color: #FFD750;
     text-align: center;
-}
\ No newline at end of file
+}
+>>>>>>> 94af5a58af6de069fa86df3ab26eb69441c2c21f
```
Comment (LEMON4DE) [inline public/styles.css]:
```suggestion
```

File: controller/controllerHelper.php [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+            $_SESSION['logged_in'] = true;
+
+        }
+        
+        $logged_in = isset($_SESSION['logged_in']) && $_SESSION['logged_in'];
+        
+        return $logged_in;
+    }
+}
```
Comment (LEMON4DE) [inline controller/controllerHelper.php]:
```suggestion
        if($validated == true){
            $_SESSION['logged_in'] = true;
        }
        
        $logged_in = isset($_SESSION['logged_in']) && $_SESSION['logged_in'];
        
        return $logged_in;
    }
}
```

**Review by LEMON4DE — CHANGES_REQUESTED** [review]

**Review by LEMON4DE — APPROVED** [review]

**Pull request merged**